urllib3/response.py

```diff
@@ -269,7 +270,18 @@ def read(self, amt=None, decode_content=None, cache_content=False):
             except HTTPException as e:
                 # This includes IncompleteRead.
                 raise ProtocolError('Connection broken: %r' % e, e)
+        except Exception:
```

Comment:
I thought about just except: but read in the python docs that it wasn't recommended. I did a search in your code and couldn't find it anywhere so opted for this. Don't all exceptions derive from Exception?